pithos/plugins/mpris.py

```diff
@@ -424,6 +420,20 @@ def _get_metadata(self, window, song):
 
         return metadata
 
+    def _metadata_equal(self, m1, m2):
```

Comment:
PyGObject should be internally using [g_variant_compare()](https://developer.gnome.org/glib/stable/glib-GVariant.html#g-variant-compare)